src/Components/index.js

```diff
@@ -0,0 +1,4 @@
+import SpotifyAuth from "./Spotify/SpotifyAuth";
+import tokenExtractor from "./Spotify/TokenExtractor"
```

Comment:
These `index.js` files are mostly for components that are used elsewhere. Because `tokenExtractor` is only used by `SpotifyAuth`, it shouldn't be re-exported.

Reply:
Okai makes sense, I thought we wanted something similar to a python package for each folder and all contents